mlflow/utils/__init__.py

```diff
@@ -6,11 +6,16 @@
 from itertools import islice
 from sys import version_info
 
+import pydantic
+from packaging.version import Version
+
 _logger = logging.getLogger(__name__)
 
 
 PYTHON_VERSION = f"{version_info.major}.{version_info.minor}.{version_info.micro}"
 
+IS_PYDANTIC_V1 = Version(pydantic.__version__).major < 2
+
```

Comment:
i'm not sure this is the best place for it, but i was running into circular import issues elsewhere